packages/composites/augur/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "src"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["dist", "**/*.spec.ts", "**/*.test.ts"],
+  "references": [{ "path": "../../core/test-helpers" }, { "path": "../../core/bootstrap" }]
```

```suggestion
  "references": [
  { "path": "../../core/test-helpers" },
  { "path": "../../core/bootstrap" },
  { "path": "../../sources/therundown" }
  ]
```
Should fix your docker build problem.

---

That did it, thanks! I'll keep this in mind for next time